src/imars3d/backend/diagnostics/rotation.py

```diff
@@ -86,6 +98,26 @@ def _find_rotation_center(
         # locate 180 degree pairs
         atol = atol_deg if in_degrees else np.radians(atol_deg)
         idx_low, idx_hgh = find_180_deg_pairs_idx(angles, atol=atol, in_degrees=in_degrees)
+        if num_pairs <= 0 or num_pairs >= idx_low.size:
+            logger.info("Using all pairs of angles")
```

Comment:
shouldn't this be a warning?

Reply:
I normally reserve warning for things that are not what the user may have intended, or are doing something slightly dangerous. Since it is controlled by an input variable I consider the number of pairs used as a bit of information.